<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModSwap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="delete_actual_mods_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete current mods folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="available_mods_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Available mods folders:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_text">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="folders_list">
     <property name="leftMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="swap_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guide_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Check guide</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
